INSTALL.md

```diff
@@ -0,0 +1,166 @@
+| ACE & TAO                       | TAO 1.3 (Optional)            | ACE & TAO website When you start your components in a networked environment, TAO can be used to set up communication between components. CORBA is used as a 'background' transport and is hidden for normal users.                                                                     |
+| Omniorb                         | 4 (Optional)                  | Omniorb website Omniorb is more robust and faster than TAO, but has less features. CORBA is used as a 'background' transport and is hidden for normal users.                                                                                                                           |
+
+All these packages are provided by most Linux distributions. In Mac OS X, you can install them easily using fink or macports. Take also a look on the Orocos.org RTT download page for the latest information.
+
+## Binary packages
+
+### Ubuntu and Debian
+
+The [Robot Operating System (ROS)](http://www.ros.org/) project provides binary packages for Ubuntu and Debian Jessie. See [REP 3 - Target Platforms](http://www.ros.org/reps/rep-0003.html) for a list of supported platforms. The dependencies of the Orocos core packages to other ROS packages are minimal. This is the recommended way to get started with the toolchain for Ubuntu and Debian users.
+
+Check out the [ROS Installation instructions](http://wiki.ros.org/ROS/Installation) for detailed instructions on how to install a ROS base system. Afterwards, the Orocos Toolchain can be installed with the command
+
+```
+sudo apt-get install ros-<distro>-orocos-toolchain
+```
+
+## Install from source
+
+### Build tools and dependencies
+
+#### Debian/Ubuntu
+
+Most dependencies mentioned above can be installed using the APT package management system:
+
+```
+sudo apt-get install build-essential git cmake libboost-all-dev libxml-xpath-perl
+```
+
+*Optional:* In case you want to build RTT with CORBA support, you also need to install the OmniORB (recommended) or TAO development packages:
+
+```
+sudo apt-get install omniorb omniidl omniorb-idl omniorb-nameserver libomniorb4-dev
+```
+
+#### Other Linux distributions
+*TODO*
+
+#### MacOS X
+*TODO*
+
+#### Windows
+*TODO*
+
+### Quick start
+
+The [orocos_toolchain](https://github.com/orocos-toolchain/orocos_toolchain) repository contains a shell script which implements all of the following steps and is the recommended way to install the Orocos toolchain from source:
+
+For Linux:
+```
+mkdir -p ~/orocos-toolchain
+cd ~/orocos-toolchain
+wget https://raw.githubusercontent.com/orocos-toolchain/orocos_toolchain/master/install.sh
+./install.sh --help
```

Comment:
The downloaded script is not executable by default, so either we prepend this with ```sh``` or we add a ```chmod +x ./install.sh``` step.